item-spec/item-spec.md

```diff
@@ -299,6 +299,13 @@ As detailed above, Items contain properties, which are the main source of metada
 extensions can add further property fields as well. Any property that can be specified for an Item can also be specified for
 a specific asset. This can be used to override a property defined in the Item, or to specify fields for which there is no
 single value for all assets.
+If a property is defined in the Item Properties, it is the default value for all assets in the Item.
+If a property is defined in a Collection on the top-level, it is the default value for all assets in the Collection.
+Assets may override the properties inherited for specific assets (see example below).
+Note that in some cases the inheritance may not lead to the expected results if other semantics have been defined for the property.
+See the examples for `gsd` and `eo:bands` below.
+It also applies to some other fields such as `created` or `updated`
+(metadata-related timestamps would get inherited and mixed with data-related timestamps that are defined in assets).
```

Comment:
This parenthetical note is confusing to me - I imagine it will be for other people as well.

```suggestion
(inheritance may be undesirable in these cases, e.g. the `created` attribute of an item would describe when the metadata was generated, whereas on an asset the `created` attribute would describe when the data file itself was created).
```